server/config/config.go

```diff
@@ -744,15 +744,19 @@ type ReplicationConfig struct {
 	LocationLabels typeutil.StringSlice `toml:"location-labels,omitempty" json:"location-labels"`
 	// StrictlyMatchLabel strictly checks if the label of TiKV is matched with LocationLabels.
 	StrictlyMatchLabel bool `toml:"strictly-match-label,omitempty" json:"strictly-match-label,string"`
+
+	// When PlacementRules feature is enabled. MaxReplicas and LocationLabels are not uesd any more.
+	EnablePlacementRules bool // Keep it false before full feature get merged. `toml:"enable-placement-rules" json:"enable-placement-rules,string"`
```

Comment:
we also need add it to `toml`?

Reply:
Comment it on purpose so neither config file nor pd-ctl can enable it.

Reply:
I think we can add a comment in toml to avoid forgetting it.